pkg/yurthub/proxy/remote/remote.go

```diff
@@ -202,3 +222,15 @@ func (rp *RemoteProxy) errorHandler(rw http.ResponseWriter, req *http.Request, e
 	}
 	rw.WriteHeader(http.StatusBadGateway)
 }
+
+func isBearerRequest(req *http.Request) bool {
+	auth := strings.TrimSpace(req.Header.Get("Authorization"))
+	if auth != "" {
+		parts := strings.Split(auth, " ")
+		klog.V(5).Infof("request: %s with bearer token: %s", util.ReqString(req), parts[1])
```

Comment:
I think we should keep this line under the if statement below.  `parts[1]` will panic If parts length < 2.

Reply:
Well, this line is generated by `go mod tidy`, otherwise we will get error when building yurthub. Maybe we depend on the pkg somewhere.
```
missing go.sum entry for module providing package github.com/mxk/go-flowrate/flowrate (imported by k8s.io/apimachinery/pkg/util/proxy); to add:
        go get k8s.io/apimachinery/pkg/util/proxy@v0.19.7
```